code/datums/ai/monkey/monkey_controller.dm

```diff
@@ -0,0 +1,211 @@
+/*
+AI controllers are a datumized form of AI that simulates the input a player would otherwise give to a mob. What this means is that these datums
+have ways of interacting with a specific mob and control it.
+*/
+///OOK OOK OOK
+
+/datum/ai_controller/monkey
+	blackboard = list(BB_MONKEY_AGRESSIVE = FALSE,\
```

Comment:
Are there any upsides to making it a list? What's the difference between making it just vars instead of list? This reminds me a lot of virus code.

Reply:
As per readme.md:
"### Blackboard
The blackboard is an associated list keyed with strings and with values of whatever you want. These store information the mob has such as "Am I attacking someone", "Do I have a weapon". By using an associated list like this, no data needs to be stored on the actions themselves, and you could make actions that work on multiple ai controllers if you so pleased by making the key to use a variable."